<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>seaerchMenu</class>
 <widget class="QDialog" name="seaerchMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SearchMenu</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>311</width>
     <height>401</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Search Menu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="searchByTeam">
       <property name="text">
        <string>Search By Team</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="searchByPlayer">
       <property name="text">
        <string>Search By Player</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="searchByGame">
       <property name="text">
        <string>Search By Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="searchStandings">
       <property name="text">
        <string>Search Standings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchNext">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="backToMainMenu">
       <property name="text">
        <string>Back to Main Menu</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
